admin/controllers/service-manager.js

```diff
@@ -13,374 +13,343 @@ const sortingAttributesService = require('../services/sorting-attributes.service
-  } catch (error) {
-    return next(error)
-  }
-  return res.redirect('/service-manager/check-settings/updated')
-}
-
-/**
- * View manage check windows.
- * @param req
- * @param res
- * @param next
- * @returns {Promise.<void>}
- */
-const getCheckWindows = async (req, res, next) => {
-  res.locals.pageTitle = 'Manage check windows'
-  req.breadcrumbs(res.locals.pageTitle)
-
-  let checkWindows
-  let checkWindowsFormatted = []
-  let checkWindowsCurrent
-  let checkWindowsPast
-
-  // Sorting
-  const sortingOptions = [
-    { 'key': 'checkWindowName', 'value': 'asc' },
-    { 'key': 'adminStartDate', 'value': 'asc' },
-    { 'key': 'checkStartDate', 'value': 'asc' }
-  ]
-  const sortField = req.params.sortField === undefined ? 'checkWindowName' : req.params.sortField
-  const sortDirection = req.params.sortDirection === undefined ? 'asc' : req.params.sortDirection
-  const { htmlSortDirection, arrowSortDirection } = sortingAttributesService.getAttributes(sortingOptions, sortField, sortDirection)
-
-  // Get current check windows
-  try {
-    checkWindows = await checkWindowDataService.sqlFindCurrentAndFutureWithFormCount(sortField, sortDirection)
-    checkWindowsCurrent = checkWindowService.formatCheckWindowDocuments(checkWindows, true, true)
-  } catch (error) {
-    return next(error)
-  }
-
-  // Get past check windows
-  try {
-    checkWindows = await checkWindowDataService.sqlFindPast(sortField, sortDirection)
-    checkWindowsPast = checkWindowService.formatCheckWindowDocuments(checkWindows, false, false)
-  } catch (error) {
-    return next(error)
-  }
-
-  // @TODO: Better merge? Object.assign overrides elements.
-  if (checkWindowsCurrent) {
-    checkWindowsCurrent.forEach((chc, index) => {
-      checkWindowsFormatted.push(chc)
-    })
-  }
-  if (checkWindowsPast) {
-    checkWindowsPast.forEach((chp, index) => {
-      checkWindowsFormatted.push(chp)
-    })
-  }
-
-  res.render('service-manager/check-windows', {
-    breadcrumbs: req.breadcrumbs(),
-    checkWindowList: checkWindowsFormatted,
-    htmlSortDirection,
-    arrowSortDirection
-  })
-}
-
-/**
- * Add window check form.
- * @param req
- * @param res
- * @param next
- * @returns {Promise.<void>}
- */
-const checkWindowsForm = async (req, res, next) => {
-  let error = ''
-  let errorMessage = ''
-  let checkWindowData = ''
-  let successfulPost = false
-  let actionName = (req.params.action === 'add' && req.params.id === undefined) ? 'Create' : 'Edit'
-  let urlActionName = req.params.action
-  let currentYear = moment.utc(Date.now()).format('YYYY')
-  let adminIsDisabled = 0
-  let checkStartIsDisabled = 0
-
-  req.breadcrumbs('Manage check windows', '/service-manager/check-windows')
-  res.locals.pageTitle = actionName + ' check window'
-  req.breadcrumbs(res.locals.pageTitle)
-
-  if (req.params.id !== undefined) {
+    let settings
+    const successfulPost = req.params.status || false
+    console.log('session:', req.session)
```

Comment:
use winston

Reply:
@GuyHarwood i don't find this log useful at all, I am removing it.